### PR TITLE
Utility Tool Additions

### PR DESCRIPTION
**Pull request by RanikOrtega**
#### Summary
Content "New Utility Tools"

#### Purpose of change
Multi-Tools and folding knives are a dime a dozen. But real life examples of specialized multi-tools for camping exist and should be made available in game.

#### Describe the solution
Created a "Camping" Multitool and Utility folding knife.

#### Testing
Debug spawned items and confirmed their functions were working. Then teleported around and until I found an instance of the item spawned naturally.
Utility Knife
Camping multitool

#### Additional context
Real life references to "Deluxe" multitools:
https://www.leatherman.com/signal-832262.html
https://www.leatherman.com/free-k4x-591.html

### CODE REVIEW

File: data/json/items/melee/swords_and_blades.json [hunk 1]
```diff
@@ -276,6 +276,26 @@
     "weapon_category": [ "KNIVES" ],
     "melee_damage": { "stab": 12 }
   },
+  {
+    "id": "utility_knife_folding",
+    "type": "TOOL",
+    "name": { "str": "utility folding knife", "str_pl": "utility folding knives" },
+    "description": "A 'Hideman' brand folding knife, this model includes extra utility options such as a screwdriver, Awl, and small pry bar.",
```
Comment (TheShadowFerret) [inline data/json/items/melee/swords_and_blades.json:283]:
```suggestion
    "description": "A 'Hideman' brand folding knife, this model includes extra utility options such as a screwdriver, awl, and small pry bar.",
```

Comment (RanikOrtega) [reply data/json/items/melee/swords_and_blades.json:283]: Fixing

**Comment by RanikOrtega** [comment]
Ready for review and to Merge I believe. Only failed test is Ubuntu and even it says that it had no errors.

**Comment by BlizzDA** [comment]
Changing the name to something legally distinct shouldn't be an issue, looking at the description. Just if you wanted to call it an actual 'Leatherman' brand tool. We have games, books, guns and medicine named that way etc.

**Comment by RanikOrtega** [comment]
> Changing the name to something legally distinct isn't required, looking at the description. Just if you wanted to call it an actual 'Leatherman' brand tool. We have games, books, guns and medicine named that way etc.

I like my puns. 😄  And just felt like it would be safer as " Hideman"

**Pull request merged**